packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -58,7 +58,9 @@ export class Runner {
                 timestamp: new Date().toJSON(),
                 error: null,
             },
-            scanResult: null,
+            scanResult: {
```

Comment:
why can't we just set scanResult only when available instead of setting it to pending?
scanResult can instead be set to undefined.

Also, Ashwin's comment was on scan notification payload containing empty string. can we fix that?

Reply:
- because in the beginning of the task, we start scanning, means scanning is pending, it might fail or pass at the end of it, but if it's null, it means it's we don't know what was the state of the scan, also the 'pending' value was put for that but was never used!

- if we set value to pending, then we will never send empty scanState in case of run failure which solves Ashwin's comment

Reply:
scanStatus should only be set when runStatus is completed. We use the runStatus for progress & scanStatus for the scan outcome.